lib/modules/manager/composer/extract.ts

```diff
@@ -127,9 +128,9 @@ export async function extractPackageFile(
               depType,
               depName,
               currentValue,
-              datasource: GithubTagsDatasource.id,
-              packageName: 'php/php-src',
-              extractVersion: '^php-(?<version>.*)$',
+              datasource: DockerDatasource.id,
+              extractVersion: '^(?<version>\\d+(?:\\.\\d+)*[^-]*)',
```

Comment:
did you validated this, so we don't get false versions?

isn't the previous git-tags datasource more safe?

Reply:
My main reason for preferring docker here is to try and help with https://github.com/renovatebot/renovate/discussions/18624. We've already seen in our CI that renovate opens PRs before the docker image is ready which causes CI to fail.

Happy to revert to github-tags since we know that's already working if you'd prefer though.

Reply:
no, using docker datasource was also my preference before 😉.

we need to make the regex as failsafe as possible. maybe we should limit to known prerelease verbs only? if we missed something, we can fix it later.